pkg/agent/qrm-plugins/network/staticpolicy/util.go

```diff
@@ -230,17 +261,18 @@ func packAllocationResponse(req *pluginapi.ResourceRequest, resourceName string,
 		ContainerIndex: req.ContainerIndex,
 		PodRole:        req.PodRole,
 		PodType:        req.PodType,
-		ResourceName:   resourceName,
+		ResourceName:   req.ResourceName,
 		AllocationResult: &pluginapi.ResourceAllocation{
 			ResourceAllocation: map[string]*pluginapi.ResourceAllocationInfo{
-				resourceName: {
+				req.ResourceName: {
```

Comment:
Is it correct to use req.ResourceName instead of the const var? It seems they have the same value.

Reply:
It's ok here. But they aren't always same.